cmd/ebrelayer/types/types.go

```diff
@@ -44,6 +44,8 @@ func (d Event) String() string {
 type EthereumEvent struct {
 	To                    []byte
 	Symbol                string
+	Name                  string
+	Decimals              int32
```

Comment:
This event should contain the token address as well.

Reply:
It does, expand down it always included the field Token common.Address